### PR TITLE
feature: add KIX region to image_uris

### PR DESCRIPTION
**Pull request by ahsan-z-khan**
*Issue #, if available:*

*Description of changes:*
Adding KIX regions account to all image uri JSONs
*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/image_uri_config/mxnet.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
                     "ap-east-1": "057415533634",
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
+                    "ap-northeast-3": "364406365360",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:22]: N/A for MXNet 0.12

File: src/sagemaker/image_uri_config/mxnet.json [hunk 2]
```diff
@@ -49,6 +50,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:53]: N/A for MXNet 1.0

File: src/sagemaker/image_uri_config/mxnet.json [hunk 3]
```diff
@@ -79,6 +81,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:84]: N/A for MXNet 11

File: src/sagemaker/image_uri_config/mxnet.json [hunk 4]
```diff
@@ -109,6 +112,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:115]: N/A for MXNet 1.2

File: src/sagemaker/image_uri_config/mxnet.json [hunk 5]
```diff
@@ -139,6 +143,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:146]: N/A for MXNet 1.3

File: src/sagemaker/image_uri_config/mxnet.json [hunk 6]
```diff
@@ -169,6 +174,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:177]: N/A for MXNet 1.4.0

File: src/sagemaker/image_uri_config/mxnet.json [hunk 7]
```diff
@@ -200,6 +206,7 @@
                         "ap-east-1": "057415533634",
                         "ap-northeast-1": "520713654638",
                         "ap-northeast-2": "520713654638",
+                        "ap-northeast-3": "364406365360",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:209]: N/A for MXNet 1.4.1 py2

File: src/sagemaker/image_uri_config/mxnet.json [hunk 8]
```diff
@@ -363,6 +374,7 @@
                     "ap-east-1": "057415533634",
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
+                    "ap-northeast-3": "364406365360",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:377]: N/A for MXNet 0.12

File: src/sagemaker/image_uri_config/mxnet.json [hunk 9]
```diff
@@ -393,6 +405,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:408]: N/A for MXNet 1.0

File: src/sagemaker/image_uri_config/mxnet.json [hunk 10]
```diff
@@ -423,6 +436,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:439]: N/A for MXNet 1.1

File: src/sagemaker/image_uri_config/mxnet.json [hunk 11]
```diff
@@ -453,6 +467,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:470]: N/A for MXNet 1.2

File: src/sagemaker/image_uri_config/mxnet.json [hunk 12]
```diff
@@ -483,6 +498,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:501]: N/A for MXNet 1.3

File: src/sagemaker/image_uri_config/mxnet.json [hunk 13]
```diff
@@ -513,6 +529,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:532]: N/A for MXNet 1.4

File: src/sagemaker/image_uri_config/mxnet.json [hunk 14]
```diff
@@ -544,6 +561,7 @@
                         "ap-east-1": "057415533634",
                         "ap-northeast-1": "520713654638",
                         "ap-northeast-2": "520713654638",
+                        "ap-northeast-3": "364406365360",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:564]: N/A for MXNet 1.4 py2

File: src/sagemaker/image_uri_config/mxnet.json [hunk 15]
```diff
@@ -704,6 +726,7 @@
                     "ap-east-1": "057415533634",
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
+                    "ap-northeast-3": "364406365360",
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:729]: N/A for MXNet 1.3 EIA

File: src/sagemaker/image_uri_config/mxnet.json [hunk 16]
```diff
@@ -734,6 +757,7 @@
...
```
Comment (saimidu) [inline src/sagemaker/image_uri_config/mxnet.json:760]: N/A for MXNet 1.4.0 EIA

File: src/sagemaker/image_uri_config/pytorch.json [hunk 17]
```diff
@@ -102,6 +102,7 @@
                     "ap-east-1": "057415533634",
                     "ap-northeast-1": "520713654638",
                     "ap-northeast-2": "520713654638",
```
Comment (shreyapandit) [inline src/sagemaker/image_uri_config/pytorch.json:104]: I don't see KIX under the EIA section here, was that intended?

Comment (ahsan-z-khan) [reply src/sagemaker/image_uri_config/pytorch.json:104]: @saimidu can you confirm if it's needed or not here?

Comment (saimidu) [reply src/sagemaker/image_uri_config/pytorch.json:104]: EIA images are only supported in a subset of AWS regions.

Reference:
- https://github.com/aws/deep-learning-containers/blob/master/available_images.md#available-deep-learning-containers-images
- https://aws.amazon.com/machine-learning/elastic-inference/pricing/

ap-northeast-3 is not one of these regions, and therefore does not need to be included.

**Review by shreyapandit — APPROVED** [review]

**Pull request merged**